src/twisted/internet/iocpreactor/iocpsupport/iocpsupport.pyx

```diff
@@ -225,7 +225,7 @@ cdef class CompletionPort:
 
         obj = None
         if ov:
-            obj, ignored = <object>ov.attached
+            obj, _ = <object>ov.attached
```

Comment:
is this change required?

Reply:
Not really.  I can remove that change if you feel it's unnecessary.

Reply:
That change is generating a big diff and the tests in this  PR are failing :) ...so I don't know if this has any wild/crazy link with the test failures :)

As long as the tests are green I am ok with this cleanup :)